guides/web-apps/sessions/README.md

```diff
@@ -127,9 +114,9 @@ JSON object can be attached to session by assigning `Data` property on `Session`
 
 ```javascript
 {
-   FirstName: "",
-   LastName: "",
-   Message: ""
+   FirstName:"",
```

Comment:
This is not valid JSON, there should be quotes around the property and there should be a space after the colon, at least according to the Google JSON style guide: https://google.github.io/styleguide/jsoncstyleguide.xml#Examples